plugins/source/aws/resources/services/frauddetector/model_versions_fetch.go

```diff
@@ -0,0 +1,32 @@
+package frauddetector
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/frauddetector"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func fetchFrauddetectorModelVersions(ctx context.Context, meta schema.ClientMeta, _ *schema.Resource, res chan<- interface{}) error {
+	c := meta.(*client.Client)
+	svc := c.Services().FraudDetector
+
+	input := new(frauddetector.DescribeModelVersionsInput)
```

Comment:
I'm surprised this isn't a child relation of the models table; don't we need to specify a model id here to get its versions? I haven't tested it, but it's fine if you have.